src/modules/Dropdown/Dropdown.js

```diff
@@ -848,12 +861,12 @@ export default class Dropdown extends Component {
   // Setters
   // ----------------------------------------
 
-  setValue = (value) => {
+  setValue = (value, resetQuery = true) => {
```

Comment:
I'm not sure that the idea with `resetQuery` is the best possible, hovewer I don't see another path, as I see in all other cases `searchQuery` should be reset.